management/account/index.md

```diff
@@ -42,17 +42,91 @@ because it is considered an identifier; you have to create a new account.
 
 ## Multi-Factor Authentication
 We highly recommend enabling the multi-factor authentication (MFA) on your account as it greatly increases its security.
+You can use a phone or other device as a virtual multi-factor authentication (TOTP) device to do this click [here](#totp-multi-factor-authentication).

```

Comment:
tady bych pridal neco jakoze mame dva zpusoby MFA overeni - totp a u2f a jednou vetou v cem kazdy z nich spociva

btw ten prvni odstavec taky tak uplne neplati - ale to teda je uz tady https://github.com/keboola/connection-docs/issues/148

Reply:
```
## Multi-Factor Authentication
We highly recommend enabling the multi-factor authentication (MFA) on your account as it greatly increases its security.

We supported two types of MFA: 
* TOTP is **software-based authentication**. When logging into a site supporting TOTP,
the authenticator app generates a six digit one-time password which users must enter in addition to their usual login details.
You can use a phone or other device as a virtual multi-factor authentication (TOTP) device to do this click [here](#totp-multi-factor-authentication).

* U2F is **a hardware device**. When signing in activate your security key,
following your security key's documentation(e.g. pressing button) rather than typing a verification code.
For using Security keys use the [FIDO U2F](https://fidoalliance.org/) standard, please click [here](#u2f-multi-factor-authentication).
```

Reply:
Ta cast `We highly recommend enabling...` mi pride ze by tam mohla zostat. Info ze je to bezpecnejsie

Reply:
Ak je to teda takto v pohode dam to na kontrolu.

Reply:
je to v poho, mergujem